docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`2240` Users now can select the supported assets from a dropdown when creating and otc trade.
```

Comment:
It's now call external trade. So `and otc` -> `an external`

Reply:
I also need to rename the components